terralab/utils.py

```diff
@@ -27,10 +27,19 @@ def wrapper(*args: str, **kwargs: str) -> Any:
         try:
             return func(*args, **kwargs)
         except ApiException as e:
+            message = "[no message body]"
```

Comment:
i know this was already here but i think this should just be empty, this will likely confuse the user more than anything else

Reply:
yeah fair, will remove